docs/mkdocs.yml

```diff
@@ -34,33 +34,56 @@ site_dir: 'generated'
 nav:
   - Welcome:
       - Landing: index.md 
-  - Overview:
-      - Readme: readme.md
-      - User guide: direct/user-guide.md
+  - What is KubeStellar?:
+      - Overview: readme.md
+#      - User guide: direct/user-guide.md
       - Architecture: direct/architecture.md
+      - Related:
+        - KubeFlex: direct/kubeflex-intro.md
+        - KubeStellar Galaxy: direct/galaxy-intro.md
       - Release-notes: direct/release-notes.md
-  - Getting Started: 
-          - Pre-reqs: direct/pre-reqs.md
+  - Using Kubestellar:
+          - Overview: direct/setup-overview.md
+          - Prerequisites: direct/pre-reqs.md
+#              - Pre-reqs: direct/pre-reqs.md
           - Setting up KubeStellar:
-              - Using existing hosting cluster: direct/hosting-cluster.md
-              - KubeStellar on KIND: direct/ks-on-kind.md
-              - KubeStellar on K3D: direct/deploy-on-k3d.md
-              - KubeStellar on OCP: direct/ks-on-ocp.md
+              - Set Up Kubestellar Core Components: # common-subs/placeholder.md
+                - Try the "Common-setup": direct/common-setup-intro.md
+                - Step by Step: direct/common-setup-step-by-step.md
+                - (experimental) Via Helm Chart: direct/core-chart.md
+#                - Prepare the environment: common-subs/placeholder.md
+#                - Initialize Kubeflex: common-subs/placeholder.md
+#                - Install Kubestellar Core components: common-subs/placeholder.md
+#                - Create the Inventory & Transport Space: common-subs/placeholder.md
+#                - Create the Workload Description Space: common-subs/placeholder.md
+              - Set up and Register Workload Execution Clusters (WECs): direct/example-wecs.md
+#              - Register WECS with core: common-subs/placeholder.md
+              - Create Workload Deployments: direct/deploy-workloads.md
```

Comment:
I do not love the "deploy" terminology here.

(A) In some sense, it is KubeStellar's job to "deploy" the workload in each WEC. What this section is trying to talk about, I think, is not that but rather putting the workload desired state into the WDS(es).

(B) What goes into the WDS is not just the workload desired state from the user, but also: the control object desired state from the user, the control object reported state from the KubeStellar implementation, and the workload reported state from the WEC(s) (originally) through KubeStellar.

Reply:
I am not wedded to a particular phrasing, but it needs to be only two or three words.
Would "commission workloads" be a better phrase?

Reply:
![image](https://github.com/kubestellar/kubestellar/assets/25445603/78f2b59b-b157-44cf-b3cb-8b92ac0f88ee)

Something like this (and this incorporates getting rid of the second overview menu item)

Reply:
changed deploy to commission

Reply:
I had expected to follow the lead of the term "Workload _Description_ Space", so say "describe". But what goes in that space is not just the description of the workload but also the control objects.

"commission" is OK. It is more general and vague, can conceivably include creating the control objects as well as the workload desired state.